jo-rdan/Free-Mentor
# #168391812 Make the user interface more responsive

#### What does this PR do?
It fixes non-responsive pages in the user interface 
#### Description of Task to be completed?
- make the login form more responsive
- make the user dashboard page more responsive
- make specific mentor page more responsive
#### How should this be manually tested?
- Please follow this [link](jo-rdan.github.io/Free-Mentor/UI) to see my progress
- Once to on the website, try to zoom out the page and the footer should stay put on the bottom
#### Any background context you want to provide?
- N/A
#### What are the relevant pivotal tracker stories?
[#168391812](https://www.pivotaltracker.com/story/show/168391812)
#### Screenshots (if appropriate)
**on the index file**
![image](https://user-images.githubusercontent.com/51836855/64598496-a9c99580-d36c-11e9-852a-632741340f23.png)

**on the user dashboard**
![image](https://user-images.githubusercontent.com/51836855/64598611-e2696f00-d36c-11e9-9f7c-c21b71e3da98.png)

**on the specific mentor page**
![image](https://user-images.githubusercontent.com/51836855/64598731-0c229600-d36d-11e9-8033-f16ff4063b79.png)


#### Questions:
